Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -210,6 +214,12 @@ open class MessageSizeCalculator: CellSizeCalculator {
         let isFromCurrentSender = dataSource.isFromCurrentSender(message: message)
         return isFromCurrentSender ? outgoingAccessoryViewPadding : incomingAccessoryViewPadding
     }
+    
+    public func accessoryViewPosition(for message: MessageType) -> AccessoryPosition {
```

Comment:
Can this be `open`

Reply:
This could not be `open`, because of `duplicate modifier` error. In addition, the method `accessoryViewSize` and `accessoryViewPadding` is not `open`, so this should not be `open` either.